gpytorch/lazy/lazy_tensor.py

```diff
@@ -1859,25 +1859,41 @@ def symeig(self, eigenvectors: bool = False) -> Tuple[Tensor, Optional["LazyTens
             pass
         return self._symeig(eigenvectors=eigenvectors)
 
-    def to(self, device_id):
+    def to(self, *args, **kwargs):
         """
-        A device-agnostic method of moving the lazy_tensor to the specified device.
+        A device-agnostic method of moving the lazy_tensor to the specified device or dtype.
+        Note that we do NOT support non_blocking or other `torch.to` options other than
+        device and dtype.
```

Comment:
Should we error out if those are passed? Or log a warning?

Reply:
They're all silent options?

Reply:
Mention that it ignores currently. Might be a good thing for linear operator to ultimately support though.